compiler+runtime/src/cpp/jank/runtime/core/seq.cpp

```diff
@@ -52,7 +52,6 @@ namespace jank::runtime
     return visit_object(
       [=](auto const typed_o) -> native_bool {
         using T = typename decltype(typed_o)::value_type;
-
```

Comment:
This line should not be removed.

Reply:
i've added it back but github has not updated the diff yet.